src/SFA.DAS.ApplyService.InternalApi/Services/Moderator/ModeratorReviewCreationService.cs

```diff
@@ -72,6 +71,21 @@ private List<ModeratorPageReviewOutcome> GenerateSectionReviewOutcomes(Guid appl
                         Comment = null
                     });
                 }
+
+                if (section.SequenceNumber == RoatpWorkflowSequenceIds.DeliveringApprenticeshipTraining && section.SectionNumber == RoatpWorkflowSectionIds.DeliveringApprenticeshipTraining.ManagementHierarchy)
+                {
+                    // Inject page to show Financial information to Moderator
```

Comment:
I'd rather you moved this into a subfunction with a name that indicates what this does rather than a comment... Or just take the comment out :)